File: android/apollo/src/main/java/io/muun/apollo/domain/model/CreateFirstSessionOk.java
package io.muun.apollo.domain.model;

import io.muun.apollo.domain.model.user.User;
import io.muun.common.crypto.hd.PublicKey;

public class CreateFirstSessionOk {

    public User user;
    public PublicKey cosigningPublicKey;
    public PublicKey swapServerPublicKey;

    /**
     * Constructor.
     */
    public CreateFirstSessionOk(User user,
                                PublicKey cosigningPublicKey,
                                PublicKey swapServerPublicKey) {
        this.user = user;
        this.cosigningPublicKey = cosigningPublicKey;
        this.swapServerPublicKey = swapServerPublicKey;
    }
}

File: android/apollo/src/main/java/io/muun/apollo/domain/model/CreateFirstSessionOk.kt
package io.muun.apollo.domain.model

import io.muun.apollo.domain.model.user.User
import io.muun.common.crypto.hd.PublicKey

data class CreateFirstSessionOk(
    val user: User,
    val cosigningPublicKey: PublicKey,
    val swapServerPublicKey: PublicKey,
    val playIntegrityNonce: String?,
)